### DISCUSSION TITLE
Cant cite, @ doesnt work

### DISCUSSION THREAD

**Discussion by Thomas204** [Zettlr/Zettlr#4967] [Q&A] [answered]
I have setup citations with Zotero as described in the official Zettlr manual, but I dont manage to get it work. I even tried with the demo citation database Zettlr provides. My problem: I even can't type the character '@' to start the citation process. I am working on a Mac with Mac OS 14.2.1 Thanks for any suggestions!

**Comment by Myarlathotep** [comment, marked as answer]
Is it that your keyboard layout doesn't have the @ symbol? If you change the keyboard language on your Mac to "English (US)", are you able to type it when you hit "shift+2"?

**Reply by Thomas204** [reply]
Thank you so much, that was it! Once I changed the keyboard language to english, it worked. Before I had a swiss german/french layout where @ is option + g on a mac, which didn't work.

**Reply by nathanlesage** [reply]
> Before I had a swiss german/french layout where @ is option + g on a mac, which didn't work.

This is a known issue that has been fixed in the most recent nightly builds of Zettlr, so from the next version on, you should be able to type an @ even on a Swiss/German Mac keyboard layout.

Reference: https://github.com/Zettlr/Zettlr/issues/4775

**Reply by Thomas204** [reply]
great!
